lib/client.js

```diff
@@ -60,10 +62,13 @@ function Client(userId, apiToken, apiSecret, options) {
 	};
 	this.rawGetRequest = function (uri) {
 		return request({
-			url  : this.prepareUrl(uri),
-			auth : {
+			url     : this.prepareUrl(uri),
+			auth    : {
 				user : apiToken,
 				pass : apiSecret
+			},
+			headers : {
+				"user-agent" : "node-bandwidth-v" + SDK_VERSION_NUMBER
```

Comment:
[Check here](https://github.com/bandwidthcom/node-bandwidth/blob/2.0-preview/lib/client.js#L48)

For how we did this is version 2. I think they're both about the same and no preference.